testutil/runner.go

```diff
@@ -429,7 +429,7 @@ func (r *Runner) SendVoteStatusMessage(channel model.Snowflake) {
 	sendMessage(r.DiscordSession, r.Handler, channel, "?votestatus")
 	r.DiscordMessagesCount++
 
-	activeVote, _ := r.ActiveVoteDataMap[channel]
+	activeVote := r.ActiveVoteDataMap[channel]
```

Comment:
TIL that accessing a missing value of a map returns the zero value of the type

Reply:
golint told me to remove the assignment to underscore, so I did. Don't trust me implicitly here